README.md

```diff
@@ -736,6 +736,26 @@ use the bleeding edge version of the logic. Then simply set the following config
 SitePrism.use_all_there_gem = true
 ```
 
+### Getting the list of missing elements
+
+If `#all_there?` returns false and you wish to get the list of missing elements for debugging purposes
+you may want to use `#elements_missing` method. It will return the names of missing elements out of
+expected_elements list
+
```

Comment:
If you do not provide a list of `expected_elements` this method will return **all** elements that are missing on the page; from those which are defined.